src/Board_test.py

```diff
@@ -0,0 +1,76 @@
+class BoardTest:
```

Comment:
A late comment, but just realised why this file doesn't get run by pytest.. it wants the name to start with `Test` instead of finish with it, so it would need to be `TestBoard`